src/Personal.cpp

```diff
@@ -350,6 +352,7 @@ void CWorkers::ReInit(const CString &TabFilename, const CString &TabFilename2) {
         try {
             CString name = strtok(Line.getData(), TabSeparator);
             if (name.length() == 0) {
+                (void)line;
```

Comment:
We are using C++17 so can't we use [[maybe_unused]] (https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)